.github/workflows/release.yml

```diff
@@ -86,6 +86,9 @@ jobs:
       with:
         path: ${{ runner.workspace }}/downloads/
 
+    - name: List Release Artifacts
```

Comment:
leftover from debugging maybe?

Reply:
Intentional in case we need to diagnose a further issue. We could certainly remove it later if there's a reason to do so, but these particular bits only run on a release publish event.